pom.xml

```diff
@@ -2132,6 +2132,7 @@
   </developers>
   <modules>
     <module>mycore-base</module>
+    <module>mycore-application-starter</module>
```

Comment:
Should not be part of mycore, as it hard to release together with all other modules. We need this to be an extra git project so that we can release it after mycore releases.

Reply:
It should.
As an application developer I NEED a parent project for my modules and applications, that uses the MyCoRe-Version with as few organizational and technical dependencies as possible. And should also be part of every MyCoRe-SNAPSHOT.